js/app.js

```diff
@@ -0,0 +1,263 @@
+'use strict'

```

Comment:
I'm curious to hear why you chose to use `use strict` – what benefits do you get from it?